newrelic/provider_test.go

```diff
@@ -90,6 +90,7 @@ func testAccPreCheck(t *testing.T) {
 
 func testAccCreateApplication(t *testing.T) {
 	app, err := newrelic.NewApplication(
+		newrelic.ConfigFromEnvironment(),
```

Comment:
This allows setting the `NEW_RELIC_HOST` env to staging for testing against our staging environment